editors/code/package.json

```diff
@@ -447,6 +452,17 @@
                         "Fill missing expressions with reasonable defaults, `new` or `default` constructors."
                     ]
                 },
+                "rust-analyzer.discoverProjectCommand": {
```

Comment:
You need to put this config before the `$generated-start` key, otherwise the tests that regen this file will fail

Reply:
ty, fixed!